README.md

```diff
@@ -34,46 +35,103 @@ snowdrop	    https://snowdrop.github.io/helm
+| [spring-boot-example-configmap](#spring-boot-example-configmap)        | Chart to deploy the Snowdrop Spring Boot ConfigMap example | [https://github.com/snowdrop/configmap-example](https://github.com/snowdrop/configmap-example) |
+| [spring-boot-example-health-check](#spring-boot-example-health-check)  | Chart to deploy the Snowdrop Spring Boot Health Check example | [https://github.com/snowdrop/health-check-example](https://github.com/snowdrop/health-check-example) |
+| [spring-boot-example-circuit-breaker](#spring-boot-example-circuit-breaker)  | Chart to deploy the Snowdrop Spring Boot Circuit Breaker example | [https://github.com/snowdrop/circuit-breaker-example](https://github.com/snowdrop/circuit-breaker-example) |
+| [spring-boot-example-messaging-queue](#spring-boot-example-messaging-queue)  | Chart to deploy the Snowdrop Spring Boot Messaging Queues example | [https://github.com/snowdrop/messaging-work-queue-example](https://github.com/snowdrop/messaging-work-queue-example) |
 
 ## Usage
 
-To use one of the available charts, part of the snowdrop repository such as `spring-boot-example-app`, you first need to generate a chart directory using the following command:
+### spring-boot-example-app
+
+This chart deploys and exposes a Spring Boot application using S2i.
+To install it, you need to execute the following command:
+
+```console
+$ helm install my-spring-boot-app snowdrop/spring-boot-example-app --set name=app --set s2i.source.repo=http://github.com/org/repo --set s2i.source.ref=main
+```
+
+When installing the chart, Openshift will trigger a S2i build to clone the GIT repository `http://github.com/org/repo` using the branch `main`, and then will execute Maven to build the application JAR to run on pods.
+
+**note**: Properties like the S2i base image are defined in the default [`values.yaml`](repository/spring-boot-example-app/values.yaml) file. You can override these values using the `--set` option.
+
+You can watch the progress of the chart deployment by executing:
 
 ```console
-$ mkdir rest-http
+$ watch oc get pods
+app-1-build     0/1     Completed   0          7m30s
+app-1-deploy    0/1     Completed   0          5m27s
+app-1-j5jk5     1/1     Running     0          5m25s
 ```
 
-Now, let's create the `Chart.yaml` file under the `rest-http/` folder with your Chart information:
+**note**: Openshift may take a bit to download the images before triggering the S2i build. 
+
+The pod with name `app-1-build` is automatically triggered by S2i to build the Spring Boot application. After the build is finished, the pod `app-1-deploy` will create the image that then will be used by the pod `app-1-j5jk5` to deploy the application.
+
+As the pods with names `app-1-build` and `app-1-deploy` are completed and also the pod `app-1-j5jk5` is running, we can now get the routes:
```

Comment:
`As the pods with names `app-1-build` and `app-1-deploy` are completed and also the pod `app-1-j5jk5` is running, we can now get the routes:` ==> `As soon as status of the pods `app-1-build` and `app-1-deploy` are `completed` and that the status of the `app-1-j5jk5` is running, we can now execute the following command get the route of the Service to access it:`

Reply:
Changed